bindings/nodejs/lib/types/block/output/output.ts

```diff
@@ -31,9 +31,13 @@ abstract class Output /*implements ICommonOutput*/ {
 
     private type: OutputType;
 
-    constructor(type: OutputType, amount: string) {
+    constructor(type: OutputType, amount: bigint | string) {
         this.type = type;
-        this.amount = amount;
+        if (typeof amount == 'bigint') {
+            this.amount = amount.toString(10);
```

Comment:
Does that mean the field type is just a string? Should it be a bigint instead? For other types, the field has been changed as well

Reply:
In the other type, the fields were serialized to hex which will be done automatically, but this one needs to be a decimal string, hence it's stored internally as a decimal string. If we make it `bigint` it will be serialized as hex and the values will be wrong.